tests/model_registry/rbac/test_mr_rbac_sa.py

```diff
@@ -0,0 +1,145 @@
+# AI Disclaimer: Google Gemini 2.5 pro has been used to generate a majority of this code, with human review and editing.
+import pytest
+from typing import Self, Dict, Any
+from simple_logger.logger import get_logger
+
+# Framework / Client specific imports
+from model_registry import ModelRegistry as ModelRegistryClient
+from mr_openapi.exceptions import ForbiddenException
+
+from utilities.constants import DscComponents, Protocols  # Need Protocols.HTTPS
+from tests.model_registry.constants import MR_NAMESPACE
+
+
+LOGGER = get_logger(name=__name__)
+
+
+# --- Helper to build client args ---
+# Based on the 'model_registry_client' fixture
+def build_mr_client_args(rest_endpoint: str, token: str, author: str) -> Dict[str, Any]:
+    """Builds arguments for ModelRegistryClient based on REST endpoint and token."""
+    try:
+        server, port = rest_endpoint.split(":")
+
+        # Conftest example uses is_secure=False - this implies TLS verification might be off
+        # Adjust if your environment requires strict TLS verification
+        return {
+            "server_address": f"{Protocols.HTTPS}://{server}",
+            "port": port,
+            "user_token": token,
+            "is_secure": False,  # Match conftest example (disable TLS verification)
+            "author": author,
+        }
+    except Exception as e:
+        LOGGER.error(f"Error parsing REST endpoint '{rest_endpoint}': {e}")
+        raise ValueError(f"Could not parse REST endpoint: {rest_endpoint}") from e
+
+
+# --- Test Class ---
+
+
+# Parametrize to ensure Model Registry component is enabled via DSC fixture
+@pytest.mark.parametrize(
+    "updated_dsc_component_state_scope_class",
+    [
+        pytest.param(
+            {
+                "component_patch": {
+                    DscComponents.MODELREGISTRY: {
+                        "managementState": DscComponents.ManagementState.MANAGED,
+                        "registriesNamespace": MR_NAMESPACE,
+                    },
+                }
+            },
+            id="enable_modelregistry_default_ns",
+        )
+    ],
+    indirect=True,
+    scope="class",
+)
+@pytest.mark.usefixtures("updated_dsc_component_state_scope_class")
+class TestModelRegistryRBAC:
+    """
+    Tests RBAC for Model Registry REST endpoint using ServiceAccount tokens.
+    """
+
+    @pytest.mark.smoke
+    @pytest.mark.usefixtures("sa_namespace", "service_account")  # Need SA and NS for token
+    def test_service_account_access_denied(
+        self: Self,
+        sa_token: str,  # Get token for the test SA
+        model_registry_instance_rest_endpoint: str,  # REST endpoint fixture
```

Comment:
Let's clean up the comments. They are distracting and you are already using really good and self explanatory names for fixtures.